core/src/main/java/org/apache/carbondata/core/scan/scanner/impl/BlockletFilterScanner.java

```diff
@@ -246,20 +242,26 @@ private BlockletScannedResult executeFilter(RawBlockletColumnChunks rawBlockletC
       dimensionRawColumnChunks[chunkIndex] =
           rawBlockletColumnChunks.getDimensionRawColumnChunks()[chunkIndex];
     }
+    //dimensionReadTime is the time required to read the data from dimension array
```

Comment:
update the comment , time taken to read the projection columns after applying filtering